java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- fixed bug where in scheduling a vhm refresh would result in a permission error for org admins
```

Comment:
Do we have a bugreport with a bugnumber for this?

Reply:
it's an issue. not bug

Reply:
oops, i fixed it last week but i didn't push it. lol